example-service.yaml

```diff
@@ -1,15 +1,30 @@
----
 apiVersion: v1
 kind: Service
 metadata:
-  name: example-nginx
+  name: nginx
   labels:
-    app: example
-    component: nginx
+    app: nginx
+  annotations:
+    kots.io/when: '{{repl not IsKurl }}'
 spec:
   type: ClusterIP
   ports:
-  - port: 80
+    - port: 80
+  selector:
+    app: nginx
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  labels:
+    app: nginx
+  annotations:
+    kots.io/when: '{{repl IsKurl }}'
+spec:
+  type: NodePort
```

Comment:
The PR description says this uses a NodePort for existing cluster, but this looks like it uses ClusterIP for existing cluster and NodePort for embedded.

Reply:
Will this work with the `Application` port?
Our docs mention it should be `ClusterIP`
https://docs.replicated.com/reference/custom-resource-application#ports

Reply:
ClusterIP for existing cluster. I think those docs are wrong

Reply:
@jdewinne clarified https://github.com/replicatedhq/replicated-docs/pull/596